content/en/docs/ops/configuration/traffic-management/multicluster/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Multi-cluster Traffic Management
+description: How to configure how traffic is distributed among clusters in the mesh.
+weight: 70
+keywords: [traffic-management,multicluster]
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+Within a multicluster mesh, [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)
```

Comment:
Is this the right place for the sameness note? Or should it live in `deployment models`? Or both?

Reply:
This should definitely go in deployment models. Maybe we need a glossary entry as well for `namespace sameness`. I'm fine duplicating the link here.